report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,246 @@
+    \item Train a new model on the remaining samples.
+    \item Repeat until model no longer improves as measured by the RMSE.
+\end{enumerate}
+
+It should be noted that our outlier removal process is conservative to avoid removing too many samples and as such, we have set our significance level $\alpha$ to 0.975.
+This means that we are willing to accept a 2.5\% chance of falsely removing a sample that is not an outlier.
+
+\subsection{ICA}\label{sec:methodology_ica}
+The ICA phase in our pipeline, which can be seen in figure \ref{fig:ica_data}, represents a mostly faithful replication of the original, with a few considered adjustments and exclusions due to incomplete details about the original implementation.
+In this section, we delve into these differences and the rationale behind our choices, maintaining a strong alignment with the overall methodology of the original pipeline.
+An illustration of the ICA phase of our pipeline can be seen in figure \ref{fig:ica_data}.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.2675\textwidth]{images/ica_data.png}
+	\caption{The ICA phase in our recreation of the pipeline.}
+	\label{fig:ica_data}
+\end{figure}
+
+\subsubsection{Data Preprocessing}\label{sec:ica_data_preprocessing}
+As previously mentioned, the shared preprocessing steps described in section \ref{sec:shared_preprocessing} are also used for the ICA phase.
+However, there are some additional preprocessing steps that are unique to the ICA phase and some differences to the original pipeline that we will describe here.
+
+In contrast to \citet{cleggRecalibrationMarsScience2017}, we do not weigh by the inverse of the instrument response function (IRF) before normalizing.
+The purpose of the IRF is to calibrate the measured signal to physical units since different pixels have different sensitivities\cite{chemcam_instrument_suite}.
+For example, pixels located at the edge of a CCS usually exhibit lower sensitivity compared to those at the center, and the optics vary in their reflective and absorptive properties at different wavelengths.
+One argument for inverting the IRF is that it introduces more noise in areas of interest on the spectrum by multiplying with a high value as part of the conversion to physical units.
+However, during a conversation with one of the original authors of \citet{cleggRecalibrationMarsScience2017}, criticism was raised against this.
+Specifically, the author pointed out that weighing by the inverse of the discards the alignment between the spectral data collected by the instrument in Los Alamos and the spectral data collected by the instrument on Mars.
+If the same instrument were used for all analyses, one could argue that the IRF (apart from the inverse square law correction) part is irrelevant, but that is not the case here; there are two different instruments --- one in Los Alamos and one on Mars.
+Based on these considerations, we have decided to not weigh by the inverse of the IRF.
```

Comment:
Need to remember to update this

Reply:
We just did now. Please read the new stuff.